src/_data/active_projects.yml

```diff
@@ -1,64 +1,63 @@
--
-  name: Open Budget
+- name: Open Budget
   image: openbudgetoakland_300x233.png
   link: http://openbudgetoakland.org/
   leader: Felicia Betancourt
   slack_id: C02R4185B
   slack_channel: openbudget
   description: Every 2 years the city of Oakland releases a budget in a spreadsheet that has nearly 20,000 rows, which does not make for ideal reading. The mission of Open Budget Oakland is to transform this mountain of data into an interactive display of charts and diagrams that is easy and even fun to use. Our MVP is live, but could benefit from improvements and new features.
--
-  name: Open Disclosure
+- name: Open Disclosure
   image: opendisclosure_300x233.png
   link: http://www.opendisclosure.io/
   leader: Elina Rubuliak, Suzanne Doran, and Tom Dooner
   slack_id: C02GKECTJ
   slack_channel: open-disclosure
   description: Open Disclosure helps Oaklanders understand the role of money in their local politics. By analyzing mandatory campaign finance disclosures for candidates seeking public office, Open Disclosure presents a user-friendly overview of who is raising money, from where, and how much. Open Disclosure is developed in partnership with the City of Oakland Public Ethics Commission.
--
-  name: Water Quality
+- name: Water Quality
   image: water_300x237.png
   link: http://water.openoakland.org/
   leader: Aaron Hans
-  slack_id: UA0S0PNSZ
+  slack_id: CBB52CFU2
   slack_channel: ca-water-challenge
   description: Approximately one million California residents currently only have access to drinking water that does not meet state and federal standards.  The California Drinking Water project provides user-friendly online queries, charts, and maps to show which water systems have current and persistent drinking water contamination violations.  Partnering with the State Water Board and the Community Water Center non-profit organization, Open Oakland volunteers are developing tool features using the State’s bi-monthly Human Right to Water water quality reporting data on cities, communities, schools, and daycares.  The online features allow government officials, advocates, researchers, and residents to more easily understand which water systems have contamination problems and identify solutions that support clean water for all Californians.
--
-  name: West Oakland Air Quality (WOAQ)
+- name: West Oakland Air Quality (WOAQ)
   image: woaq_300x155.png
-  link: 
+  link:
   leader: Jess Sand
-  slack_id: 
+  slack_id: C95U7371C
   slack_channel: woaq
   description: We are creating a digital platform for West Oakland residents to collect, visualize, and act on the air quality data which affect them. We are partnered with the West Oakland Environmental Indicators Project, a community group who has collected Air Quality data from their neighborhood.
--
-  name: OUSD Budget
-  image: 
-  link: 
+- name: OUSD Budget
```

Comment:
Do we want this page to be alphabetical? Projects up to WOAQ are alphabetized.
<img width="300" src="https://user-images.githubusercontent.com/20404311/62180540-424c0f00-b305-11e9-86f7-3b6acf4a546c.jpg"/>

Reply:
I dunno, do we? I think the projects up to WOAQ came from WordPress and the next three have been added after export. Thoughts?